contents/docs/feature-flags/testing.mdx

```diff
@@ -0,0 +1,50 @@
+---
+title: Testing your feature flag
+sidebar: Docs
+showTitle: true
+availability:
+    free: none
+    selfServe: full
+    enterprise: full
+---
+
+Once you've written your code, it's a good idea to test that each variant behaves as you'd expect. There are 3 ways you can do this:
+
+## Method 1: Assign an optional override
+
+You can assign a user a specific flag value by **adding an optional override** to your [release conditions](/docs/feature-flags/creating-feature-flags#release-conditions). For example, you can create an override to assign a user a specific flag value if their email is your own (or someone in your team). 
+
+To do this:
+
+1. Go to your feature flag.
+2. Ensure the feature flag is enabled by checking the "Enable feature flag" box.
+3. Add a new condition set with the condition to `email = your_email@domain.com`. Set the rollout percentage for this set to 100%.
+   
+   - in cases where `email` is not available (such as when your users are logged out), you can use a parameter like `utm_source` and append `?utm_source=your_variant_name` to your URL.
+
+
+4. Set the optional override for the variant you'd like to assign these users to.
+5. Click "Save".
+
+![How to test your feature flag](../../images/features/feature-flags/override-feature-flag.mp4)
+
+## Method 2: Use `posthog.featureFlags.override()`
+
+> **Note:** The `posthog.featureFlags.override()` method is only available in the [JavaScript web](/docs/libraries/js) and [React Native](/docs/libraries/react-native) SDKs.
+
+You can add a manual override directly in your code by calling `posthog.featureFlags.override()`:
+
+```javascript
+posthog.feature_flags.override({
```

Comment:
is it `posthog.featureFlags.override` or `posthog.feature_flags.override`?

Reply:
former, nice catch :)